src/commonmark/en/content/user/managing-dashboards.md

```diff
@@ -230,19 +230,19 @@ report, and event chart items. Click on the interpretations button
 ![](resources/images/dashboard/dashboard-interpretations-button.png),
 and the item will be expanded vertically underneath to show the
 interpretations and replies. You can like an interpretation, reply to an
-interpretation, and add your own interpretation. You can edit or delete
+interpretation, and add your own interpretation. You can edit, share or delete
 your own interpretations and replies, and if you have moderator access,
 you can delete others’ interpretations.
 
 It is possible to format the text with **bold**, *italic* by using the
 Markdown style markers \* and \_ for **bold** and *italic* respectively.
+New interpretations have a toolbar in its texfield for adding rich text.
```

Comment:
Instead of "its textfield" write "the text field"

Reply:
Done! I also missed a 't' there. I've split up the word